### PR TITLE
[4.0] Extension updated infourl

### PR DESCRIPTION
**Pull request by brianteeman**
The check of `isset` is wrong and it should be `!empty`

If you have an outdated extension then it will display a non clickable url to the xml AND if set a clickable url to a details page

The existing code would still output a `br` and and empty `a` if there was no details page available. This simple PR fixes that

### CODE REVIEW

File: administrator/components/com_installer/tmpl/update/default.php [hunk 1]
```diff
@@ -137,7 +137,7 @@
 									<td class="d-none d-md-table-cell">
 										<span class="break-word">
 										<?php echo $item->detailsurl; ?>
-											<?php if (isset($item->infourl)) : ?>
+											<?php if (!empty($item->infourl)) : ?>
 												<br>
 												<a href="<?php echo $item->infourl; ?>" target="_blank" rel="noopener noreferrer"><?php echo $this->escape($item->infourl); ?></a>
```
Comment (Quy) [inline administrator/components/com_installer/tmpl/update/default.php:142]: `$item->infourl` is escaped in anchor text but not in `href`. Separate PR?

Comment (brianteeman) [reply administrator/components/com_installer/tmpl/update/default.php:142]: yes please because that code might be elsewhere

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 36bde0567cfecbeff61d205dd6570154d9b7d778<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25494">issues.joomla.org/tracker/joomla-cms/25494</a>.</sub>

**Comment by tonypartridge** [comment]
I have tested this item :white_check_mark: successfully on acf95b0c5b40746823bb8074cd21850e560454bf<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25494">issues.joomla.org/tracker/joomla-cms/25494</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25494">issues.joomla.org/tracker/joomla-cms/25494</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**